src/__tests__/extensions/dead-clicks-autocapture.test.ts

```diff
@@ -0,0 +1,70 @@
+import { PostHog } from '../../posthog-core'
+import { assignableWindow } from '../../utils/globals'
+import { createPosthogInstance } from '../helpers/posthog-instance'
+import { uuidv7 } from '../../uuidv7'
+import { DeadClicksAutocapture } from '../../extensions/dead-clicks-autocapture'
+
+describe('DeadClicksAutocapture', () => {
```

Comment:
this outer wrapper delegates entirely to the lazy loaded one so we can test it interacts with a mock as desired and ignore the behaviour